hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/pipelines/pipelines.tsx

```diff
@@ -111,7 +111,15 @@ const COLUMNS = [
     dataIndex: 'datanodes',
     key: 'datanodes',
     isSearchable: true,
-    render: (datanodes: string[]) => <div>{datanodes.map(datanode => <div key={datanode}>{datanode}</div>)}</div>
+    render: (datanodes: string[]) => <div> {datanodes && datanodes.map(datanode =>
+      <div key={datanode.hostName}>
+            <div className='uuidtooltip'>
+            <Tooltip placement='top' title={`UUID - ${datanode && datanode.uuid}`} getPopupContainer={(triggerNode) => triggerNode}>
```

Comment:
```suggestion
            <Tooltip placement='top' title={`UUID: ${datanode && datanode.uuid}`} getPopupContainer={(triggerNode) => triggerNode}>
```

Reply:
Completed Chaanges Suggested in Review by @dombizita